k8s/ietfweb/local.py

```diff
@@ -0,0 +1,114 @@
+# Copyright The IETF Trust 2007-2024, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+from email.utils import parseaddr
+import os
+
+def _multiline_to_list(s):
+    """Helper to split at newlines and conver to list"""
+    return [item.strip() for item in s.split("\n")]
+
+
+DEFAULT_FROM_EMAIL = "donotreply@ietf.org"
+SERVER_EMAIL = "donotreply@ietf.org"
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = os.environ.get("IETFWWW_EMAIL_HOST", "localhost")
+EMAIL_PORT = int(os.environ.get("IETFWWW_EMAIL_PORT", "2025"))
+
+# Default to "development". Production _must_ set IETFWWW_SERVER_MODE="production" in the env!
+SERVER_MODE = os.environ.get("IETFWWW_SERVER_MODE", "development")
+
+# Secrets
+_SECRET_KEY = os.environ.get("IETFWWW_DJANGO_SECRET_KEY", None)
+if _SECRET_KEY is not None:
+    SECRET_KEY = _SECRET_KEY
+else:
+    raise RuntimeError("IETFWWW_DJANGO_SECRET_KEY must be set")
+
+
+_csrf_trusted_origins_str = os.environ.get("IETFWWW_CSRF_TRUSTED_ORIGINS")
+if _csrf_trusted_origins_str is not None:
+    CSRF_TRUSTED_ORIGINS = _multiline_to_list(_csrf_trusted_origins_str)
+
+FILE_UPLOAD_PERMISSIONS = 0o664
+_WAGTAILADMIN_BASE_URL = os.environ.get("WAGTAILADMIN_BASE_URL", None)
```

Comment:
`_WAGTAILADMIN_BASE_URL` all lower-case to be consistent with others?

Reply:
There's a mix really. Addressed with https://github.com/ietf-tools/www/pull/453/commits